app/views/alchemy/_preview_mode_code.html.erb

```diff
@@ -0,0 +1,4 @@
+<% if @preview_mode %>
+  <%= javascript_tag("Alchemy = { locale: '#{session[:alchemy_locale]}' };") %>
```

Comment:
Maybe we don't need the costly rails helper here? A simple `<script>` should do it, no?